lib/config.js

```diff
@@ -15,11 +15,12 @@ var path = require('path');
 /**
  * Get configuration for a project, falling back to default configuration if
  * project does not provide a configuration of its own
- * @param {String} root - root path of project
+ * @param {Object} project - The Project model tied to the package.json
  * @returns {Configuration} configuration to use for project
  */
-function config(root) {
-  var configFile = path.join(root, 'config', 'coverage.js');
+function config(project) {
```

Comment:
I'd prefer to continue to pass in the config path, rather than pass around the `project` (I'd like to keep usage of ember-cli api contained to index.js as much as possible)